anta/models.py

```diff
@@ -250,8 +255,23 @@ def supported(self) -> bool:
         """
         if not self.collected and not self.error:
             msg = f"Command '{self.command}' has not been collected and has not returned an error. Call AntaDevice.collect()."
+
             raise RuntimeError(msg)
-        return not any("not supported on this hardware platform" in e for e in self.errors)
+        return all("not supported on this hardware platform" not in e for e in self.errors)
+
+    @property
+    def returned_known_eos_error(self) -> bool:
+        """Return True if the command returned a known_eos_error on the device, False otherwise.
+
+        RuntimeError
+            If the command has not been collected and has not returned an error.
+            AntaDevice.collect() must be called before this property.
+        """
+        if not self.collected and not self.error:
+            msg = f"Command '{self.command}' has not been collected and has not returned an error. Call AntaDevice.collect()."
+            raise RuntimeError(msg)
+        known_eos_errors = ["BGP inactive", "VRF '.*' is not active", ".* does not support IP"]
```

Comment:
Should we put this in a separate module? Might be easier to maintain/access if we need to add stuff later.

Reply:
I vote when the time comes we do it :p

Reply:
The time has come! (IS-IS) :P